project/src/main/webapp/index.html

```diff
@@ -11,5 +11,12 @@
     <title>Voter Central Homepage</title>
   </head>
   <body>
+    <h2>Voter Central Home Page</h2>
+    <form id="location-input" action="/data" method="POST">
+      <label for="zipcode">Enter your zipcode:</label>
+      <br>
+      <input type="text" id="zipcode"/>
```

Comment:
- Add the `required` attribute to make sure users have filled in the zip code before submitting
- Perhaps switch the type to "number" for zipcode
- Perhaps set the `placeholder`attribute (to something like "00000")
(In the future, it might be nice to allow users to enter a street address in addition to the zipcode.)

Reply:
Makes sense. I've changed it to be a number for now.